app/services/katello/ui_notifications/pulp/proxy_disk_space.rb

```diff
@@ -5,7 +5,9 @@ class ProxyDiskSpace
         class << self
           def deliver!
             SmartProxy.unscoped.with_content.each do |proxy|
-              percentage = proxy.statuses[:pulp].storage['pulp_dir']['percent']
+              status = proxy.statuses[:pulp] || proxy.statuses[:pulpnode]
+              next unless (percentage = status.storage.dig('pulp_dir', 'percent'))
```

Comment:
hah, Hash#dig is a new one to me!  /me adds it to his toolbox

Reply:
do you think it could be possible for status to be nil here?  Should we handle that condition?

Reply:
Yea dig is great!

I suppose status could be, updated to use the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/).